src/ScottPlot5/ScottPlot/Range.cs

```diff
@@ -73,5 +73,36 @@ public double Clamp(double value)
 
             return value;
         }
+
+        public static Range GetRange(double[,] input)
```

Comment:
This can just be implemented simpler:
```cs
public static Range GetRange(double[,] input) => GetRange(input.Cast<double>());
```

`T[,]` implements `IEnumerable` but not `IEnumerable<T>`, hence the cast. However, since the overload which takes `IEnumerable<double>` doesn't create a concrete version of the stream (i.e. it doesn't call `ToArray()` or similar) I don't think the performance difference should be noticeable above the (minor) performance drop that comes from using `IEnumerable` compared to direct subscripting.

If you think that whatever performance difference there is is too high then fair enough, this isn't that much code to duplicate, I just thought I'd bring up a simpler option.